test/api.ts

```diff
@@ -35,33 +32,13 @@ export class Api {
     };
   }
 
-  put(data) {
+  put(data: string | object) {
     return { to: (url: string) => request(this.app).put(url).send(data) };
   }
 
-  putMultipartForm(data: FormData[], files = []) {
```

Comment:
this is only needed if we have forms on the page; it's not applicable for us

Reply:
Makes sense, happy to remove it now or later